ocs_ci/ocs/ui/base_ui.py

```diff
@@ -55,6 +56,54 @@
 logger = logging.getLogger(__name__)
 
 
+def wait_for_element_to_be_clickable(locator, timeout=30):
+    """
+    Wait for an element to be clickable.
+
+    Args:
+        locator (tuple): A tuple containing the locator strategy (e.g., By.ID, By.XPATH) and the locator value.
+        timeout (int): Maximum time (in seconds) to wait for the element to be clickable. Defaults to 30 seconds.
+
+    Returns:
+        selenium.webdriver.remote.webelement.WebElement: The clickable web element.
+
+    """
+    wait = WebDriverWait(SeleniumDriver(), timeout)
+    try:
+        web_element = wait.until(
+            ec.visibility_of_element_located((locator[1], locator[0]))
```

Comment:
shouldn't we use `element_to_be_clickable()` instead of `visibility_of_element_located()`?